shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -991,6 +1151,9 @@ - (void)setMarkedText:(NSString*)markedText selectedRange:(NSRange)markedSelecte
   NSRange selectedRange = _selectedTextRange.range;
   NSRange markedTextRange = ((FlutterTextRange*)self.markedTextRange).range;
 
+  if (_scribbleInProgress || _scribbleFocusStatus != FlutterScribbleStatusUnfocused)
```

Comment:
You need curly braces here (https://google.github.io/styleguide/objcguide.html#conditionals)